Assets/_BForBoss/_Weapons/Scripts/Weapons/AutomaticWeaponBehaviour.cs

```diff
@@ -17,13 +15,15 @@ protected override void OnFireInputAction(InputAction.CallbackContext context)
             if (context.canceled)
             {
                 _isFiring = false;
+                _timeSinceFire = 0;
```

Comment:
No change needed:
Iteration two will need 
`_timeSinceFire -= Time.deltaTime;`

Reply:
Will note that down when planning next iteration